src/main/java/com/cburch/logisim/std/ttl/Ttl7447HdlGenerator.java

```diff
@@ -42,50 +44,63 @@ public Ttl7447HdlGenerator() {
 
   @Override
   public LineBuffer getModuleFunctionality(Netlist TheNetlist, AttributeSet attrs) {
-    return LineBuffer.getBuffer()
-        .add("""
-            Sega  <= segments(0);
-            Segb  <= segments(1);
-            Segc  <= segments(2);
-            Segd  <= segments(3);
-            Sege  <= segments(4);
-            Segf  <= segments(5);
-            Segg  <= segments(6);
+    final var contents = LineBuffer.getHdlBuffer();
```

Comment:
Please move this line closer to the first use (after setting up `decoder`, line 65?)

Reply:
Done